app/jobs/transcode_cache_clean_job.rb

```diff
@@ -0,0 +1,7 @@
+class TranscodeCacheCleanJob < ApplicationJob
+  queue_as :transcode_cache_cleaner
+
+  def perform
+    TranscodedItem.where.not(last_used: (3.days.ago..)).destroy_all
```

Comment:
Isn't three days rather long? Since we only run this every 24h, this means that a file can exist up to 96h.

Reply:
I was thinking about making this configurable, if someone else wants to do a deploy. I'll do that. For us three days is definitely fine, since we have quite a lot of space left. Transcoded files are also usually a lot smaller than their source material.

Reply:
👍